websites/U/UpBeatRadio/dist/metadata.json

```
@@ -7,7 +7,8 @@
   "description": {
     "en": "UpBeat is an ad-free, community powered online radio.",
     "nl": "UpBeat is een advertentievrije, community aangedreven online radio.",
-    "ga_IE": "Is raidió ar líne saor in aisce, faoi thiomáint ag an bpobal é UpBeat."
+    "ga_IE": "Is raidió ar líne saor in aisce, faoi thiomáint ag an bpobal é UpBeat.",
+    "de": "UpBeat ist ein werbefreies, Community betriebenes online radio.
```

```suggestion
    "de": "UpBeat ist ein werbefreies, Community betriebenes online Radio."
```
Radio = noun = capitalize